django-debug-toolbar/meta.yaml

```diff
@@ -10,6 +10,9 @@ source:
 build:
     number: 1
 
+extra:
```

Comment:
This won't do anything as far as I'm aware. The skip section is in the ``build`` section, and I need to update ObviousCI to listen to the value...

https://github.com/pelson/Obvious-CI/issues/32 refers